### PR TITLE
Add pangram

### PR DESCRIPTION
**Pull request by BNAndras**
Another earlier #48in24 exercise

### CODE REVIEW

File: exercises/practice/pangram/.meta/config.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "files": {
+    "solution": [
+      "src/lib.cairo"
+    ],
+    "test": [
+      "tests/pangram.cairo"
+    ],
+    "example": [
+      ".meta/example.cairo"
+    ]
```
Comment (0xNeshi) [inline exercises/practice/pangram/.meta/config.json:14]: Missing invalidator

File: exercises/practice/pangram/.meta/example.cairo [hunk 2]
```diff
@@ -0,0 +1,28 @@
+pub fn is_pangram(sentence: ByteArray) -> bool {
+    let mut seen: Felt252Dict<bool> = Default::default();
+
+    let mut i = 0;
+    while i < sentence
+        .len() {
+            let mut chr = sentence[i];
```
Comment (0xNeshi) [inline exercises/practice/pangram/.meta/example.cairo:7]:
```suggestion
            let chr = sentence[i];
```
nit: `chr` does not need to be mutable

**Review by 0xNeshi — CHANGES_REQUESTED** [review]

File: exercises/practice/pangram/.meta/config.json (hunk 1 above)
Comment (BNAndras) [reply exercises/practice/pangram/.meta/config.json:14]: Strange that configlet create doesn't add the invalidator file.

Comment (0xNeshi) [reply exercises/practice/pangram/.meta/config.json:17]: I agree, it's probably a missing feature in the script

**Review by 0xNeshi — APPROVED** [review]
Excellent!

**Pull request merged**